src/main/lombok/org/heigit/ohsome/ohsomeapi/config/SwaggerConfig.java

```diff
@@ -174,15 +177,6 @@ private ApiInfo apiInfo() {
     globalOperationParams.add(new ParameterBuilder().name("bpolys")
         .description(ParameterDescriptions.BPOLYS).modelRef(new ModelRef(string))
         .parameterType(query).defaultValue("").required(false).build());
-    globalOperationParams.add(new ParameterBuilder().name("types")
-        .description(ParameterDescriptions.DEPRECATED_USE_FILTER).modelRef(new ModelRef(string))
-        .allowMultiple(true).parameterType(query).defaultValue("").required(false).build());
-    globalOperationParams.add(new ParameterBuilder().name("keys")
-        .description(ParameterDescriptions.DEPRECATED_USE_FILTER).modelRef(new ModelRef(string))
-        .parameterType(query).defaultValue("").required(false).build());
-    globalOperationParams.add(new ParameterBuilder().name("values")
-        .description(ParameterDescriptions.DEPRECATED_USE_FILTER).modelRef(new ModelRef(string))
-        .parameterType(query).defaultValue("").required(false).build());
```

Comment:
This would remove these parameters from the Swagger visualization, but not from the swagger JSON? I'd prefer to keep them (as deprecated) for consistency. Otherwise this would be a breaking change, imho.

Reply:
or: if you want to change them to “hidden” parameters, I would remove them from the docs as well

Reply:
we'll hide them from the docs in a future release